src/api/app/controllers/webui/staging/workflows_controller.rb

```diff
@@ -35,8 +35,8 @@ def show
     @more_unassigned_requests = @staging_workflow.unassigned_requests.count - @unassigned_requests.size
     @ready_requests = @staging_workflow.ready_requests.first(5)
     @more_ready_requests = @staging_workflow.ready_requests.count - @ready_requests.size
-    @ignored_requests = @staging_workflow.ignored_requests.first(5)
-    @more_ignored_requests = @staging_workflow.ignored_requests.count - @ignored_requests.size
+    @excluded_requests = @staging_workflow.excluded_requests.first(5)
```

Comment:
Might be a matter of taste... I would do the split right where it's needed instead of carrying around variables (I mean the more_* ones) for it.

Reply:
again, that was already there. The purpose of this PR is not to refactor existing code.

Reply:
Oh, right. I thought `@more_ignored_requests` was similar to what we have in the collapsible. Should have checked more properly.